docs/getting-started/threat-intel-integrations.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[es-threat-intel-integrations]]
+= Enable threat intelligence integrations
+
+The Threat Intelligence view gives you a streamlined way to collect threat intelligence data that you can use for threat detection and matching. You can connect to threat intelligence sources using an <<agent-ti-integration, {agent} integration>>, the <<ti-mod-integration, Threat Intel module>>, or a <<custom-ti-integration, custom integration>>.
+
+NOTE: The warning message about remaining threat intelligence sources is removed once you’ve enabled all available {elastic-sec} threat intelligence integrations.
+
+[role="screenshot"]
+image::images/threat-intelligence-view.png[width=90%][height=90%][Shows the Threat Intelligence view on the Overview page]
+
+[float]
+[[agent-ti-integration]]
+== Enable an {agent} integration
+
+To set up an {agent} integration:
+
+. Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet} managed {agent}] on the servers you want to monitor.
+. In the Threat Intelligence view, click *Enable sources*.
+. Select an integration, and complete the installation steps.
+. Return to the Threat Intelligence view on the Overview page. Refresh the page if indicator data isn't displaying.
+
+[float]
+[[ti-mod-integration]]
+== Enable a Threat Intel module integration
+
+To set up an integration through the Threat Intel module:
+
+. Set up the {filebeat-ref}/filebeat-installation-configuration.html[{filebeat} agent] and enable the Threat Intel module.
++
+NOTE: For more information about enabling available threat intelligence filesets, refer to {filebeat-ref}/filebeat-module-threatintel.html[Threat Intel module].
+
+. Update the `securitySolution:defaultThreatIndex` <<update-threat-intel-indices, advanced setting>> by adding the appropriate index pattern name after the default {fleet} threat intelligence index pattern (`logs-ti*`):
+.. If you're using {filebeat} version 8.x _only_, add the appropriate {filebeat} threat intelligence index pattern, for example, `filebeat-8*` or `filebeat*`.
+.. If you're using a previous version of Filebeat _and_ a current one, differentiate between the threat intelligence indices by using unique index pattern names. For example, if you’re using {filebeat} version 7.0.0 and 8.0.0, add `filebeat-7*` and `filebeat-8*`.
+. Return to the Threat Intelligence view on the Overview page. Refresh the page if indicator data isn't displaying.
+
+[float]
+[[custom-ti-integration]]
+== Enable a custom integration
+
+To enable a custom integration:
+
+. Set up a way to <<ingest-data, ingest data>> into your system.
+. Update the `securitySolution:defaultThreatIndex` <<update-threat-intel-indices, advanced setting>> by adding the appropriate index pattern name after the default {fleet} threat intelligence index pattern (`logs-ti*`), for example, `custom-ti-index*`.
```

Comment:
The example is a little confusing to me. If the user has to add something to the end of the default index pattern, which is `logs-ti*`, shouldn't the example _include_ that default pattern plus something else?

Reply:
Good point! I'll update these examples.